src/main/java/de/doubleslash/keeptime/view/ViewController.java

```diff
@@ -317,6 +340,30 @@ private void initialize() {
 
    }
 
+   public String getSvgPathWithXMl(RESOURCE resource) throws ParserConfigurationException, IOException, SAXException {

```

Comment:
We need extract this method somewhere, where we can use it also from other Controllers (like ReportController).